kotlin-extensions/src/main/kotlin/kotlinx/js/JsSet.kt

```diff
@@ -2,12 +2,15 @@ package kotlinx.js
 
 @JsName("Set")
 external class JsSet<T> {
+    constructor()
+    constructor(iterable: JsIterator<T>)
```

Comment:
<img width="463" alt="Screenshot 2022-02-13 at 03 37 16" src="https://user-images.githubusercontent.com/9942723/153733425-ea6aac76-9462-4f91-9ebd-3b3310f61af9.png">

Reply:
Could I get more context on what is it and where it's from (if possible, of course)? :)

Reply:
Ok, I think I've understood what you want... Should I left `constructor(iterable: JsIterator<T>)` (I don't know if it's needed  for some kind of vanilla JS compatibility....)

Reply:
I'm not sure that it will work with `JsIterator<T>`

Hint :)
```kotlin
external class JsSet<T>(
    values: ReadonlyArray<T>? = definedExternally,
)
```

Reply:
I think you misunderstood me. I've got that you want me to create `constructor(values: ReadonlyArray<T>?)` (or, at least, I hope I understood you correctly :) ), I'm asking if I should keep vanilla JS Set constructor with iterable after that :)
So final looks of code:
``` kotlin
@JsName("Set")
external class JsSet<T> {
    constructor()
    constructor(iterable: JsIterator<T>) // Should I keep this line among others?
    constructor(values: ReadonlyArray<T>?)
    // ...
 }
 ```
 P. S. I hope I understood your screenshot...

Reply:
Latest please, with following signature
`constructor(values: JsIterator<T>?)`